Use HTTP Webserver

krishnans2006 opened the pull request:
Uses Flask HTTP Webserver to serve images, instead of TCP SocketIO connection

krishnans2006 committed:
Fix requirements

krishnans2006 committed:
Rewrite code to use HTTP webserver instead of SocketIO

pizzalemon commented on main.py:
```diff
@@ -1,141 +1,107 @@
+@app.route("/image/<int:image_id>")
+def image(image_id):
+    if image_id > last_image:
+        return {"result": "Image not found"}
```
You should probably make this return 404 or something rather than that.

pizzalemon replied:
Same thing goes for the other routes.

krishnans2006 committed:
Hotfix(es)

krishnans2006 committed:
Flask fixes + Reformat

 - Use threaded version of Flask instead of running Flask in a thread, to avoid connection refused errors
 - Stop using global variables in Flask routes (they do not work)

pizzalemon reviewed (approved):
Looks perfect!
